### ISSUE TITLE
Voiceover doesn't announce select item changes

### ISSUE THREAD

**Issue by marcospassos** [ariakit/ariakit#1647]
### Current behavior

Selecting/unselecting an item isn't announced in a multi-select combobox:

https://user-images.githubusercontent.com/943036/180036220-b6de0b5c-3414-44de-afc9-9ebcaffb300b.mov

Related:
https://twitter.com/marcospassos/status/1549773404147957762?s=20&t=mn3gcKFknwcbvJmjwZZ9aw

### Steps to reproduce the bug

1. Open sandbox: https://codesandbox.io/s/focused-panini-okjx5b?file=/src/select-combobox.tsx
2. Select and unselect an item
3. Notice nothing is announced

### Expected behavior

Announcing the changes.

### Workaround

https://github.com/ariakit/ariakit/issues/1647#issuecomment-1190581817

**Comment by diegohaz** [comment]
Thanks, @marcospassos! Could you please specify in the issue which browsers you tested? [VoiceOver is generally used with Safari](https://webaim.org/projects/screenreadersurvey9/#browsercombos), so we'll prioritize it.

Safari has historically had some issues with combo boxes. But talking specifically about a [multi-selectable combobox](https://ariakit.org/examples/combobox-multiple) (without the Select component), the announcements are fixed if we include the `aria-multiselectable` attribute on the `ComboboxPopover` component (we should fix the example).

**Comment by marcospassos** [comment]
I'm testing on Chrome. But I've also faced issues on Safari.

**Comment by marcospassos** [comment]
> Safari has historically had some issues with combo boxes. But talking specifically about a [multi-selectable combobox](https://ariakit.org/examples/combobox-multiple) (without the Select component), the announcements are fixed if we include the `aria-multiselectable` attribute on the `ComboboxPopover` component (we should fix the example)

But removing the `aria-multiselectable` introduces [other issues](https://github.com/ariakit/ariakit/issues/1549).

**Comment by diegohaz** [comment]
> > Safari has historically had some issues with combo boxes. But talking specifically about a [multi-selectable combobox](https://ariakit.org/examples/combobox-multiple) (without the Select component), the announcements are fixed if we include the `aria-multiselectable` attribute on the `ComboboxPopover` component (we should fix the example)
> 
> But removing the `aria-multiselectable` introduces [other issues](https://github.com/ariakit/ariakit/issues/1549).

You mean <ins>including</ins>? I'm talking specifically about the `ComboboxPopover` component (without Select). The [example](https://ariakit.org/examples/combobox-multiple) is lacking `aria-multiselectable`.

**Comment by marcospassos** [comment]
> You mean including? I'm talking specifically about the `ComboboxPopover` component (without Select). The [example](https://ariakit.org/examples/combobox-multiple) is lacking `aria-multiselectable`.

What boggled me is that it was being [always added previously](https://github.com/ariakit/ariakit/issues/1549). The fix removed it, and I didn't notice.

> You have to add `aria-multiselectable` to `ComboboxList` and invert the order of `ComboboxItem` and `SelectItem` (similarly to the [combobox-multiple example](https://ariakit.org/examples/combobox-multiple)). This is necessary because combobox doesn't have the concept of multi-selectable.

I can confirm adding `aria-multiselectable` fixes the issue.

**Comment by diegohaz** [comment]
The example has been fixed.
